### PR TITLE
retail z_scene_table.c OK

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/code/z_scene_table.c [hunk 1]
```diff
@@ -1633,7 +1652,7 @@ void (*sSceneDrawConfigs[SDC_MAX])(PlayState*) = {
 };
 
 void Scene_Draw(PlayState* play) {
-    if (R_HREG_MODE == HREG_MODE_SCENE_CONFIG) {
+    if ((R_HREG_MODE == HREG_MODE_SCENE_CONFIG) && OOT_DEBUG) {
```
Comment (fig02) [inline src/code/z_scene_table.c:1655]: fairly sure OOT_DEBUG first works, at least it did when i started this file the other day

Comment (engineer124) [reply src/code/z_scene_table.c:1655]: Having it first matches retail, but then breaks debug on my end. Did you check debug as well when you did it?
image

Comment (fig02) [reply src/code/z_scene_table.c:1655]: probably not, no. that sucks

**Review by fig02 — APPROVED** [review]
all the if 1's are sus, but they dont seem to be all in consistently the same place?

**Review by cadmic — APPROVED** [review]

**Pull request merged**